test/integration/editor/index.spec.ts

```diff
@@ -152,8 +152,8 @@ test('must correctly size new layout columns', async ({ page }) => {
   const secondFullWidthBoundingBox3 = await getNthFullWidthBoundingBox(1);
   const thirdFullWidthBoundingBox3 = await getNthFullWidthBoundingBox(2);
 
-  expect(firstFullWidthBoundingBox3!.width).toBe(secondFullWidthBoundingBox3!.width);
```

Comment:
I think this is because of minor differences in the resize handler widths: 
![Screenshot 2023-08-18 at 3 59 32 PM](https://github.com/mui/mui-toolpad/assets/19550456/9d40c631-f81d-49c7-b4e0-02434c45ccfd)

Sufficiently minor to ignore in my opinion